server/src/main/java/org/tctalent/server/service/db/CandidateExamService.java

```diff
@@ -46,4 +46,7 @@ CandidateExam createExam(
      * @return List of updated candidate exam records
      */
     CandidateExam updateCandidateExam(UpdateCandidateExamRequest request);
+
+    List<CandidateExam> list(long id);
+
```

Comment:
I don't think that these method can return nulls. It is a good idea to annotate them as @NotNull. Similarly if things can be null as @Nullable. That allows Intellij to do some good null checking - highlighting where null pointer exceptions could occur.

Reply:
Done